WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -3137,6 +3142,17 @@ extension AztecPostViewController: WPMediaPickerViewControllerDelegate {
     }
 }
 
+// MARK: - Accessibilty Helpers
+//
+extension UIImage {
```

Comment:
We should move the accessibility code to Aztec (the library) so that other Apps can take advantage of it as well.

Reply:
I think we can move ahead with this, as it is now, and evaluate an alternative afterwards.

:+1: